tools/ocean/argo_getdata.xml

```diff
@@ -0,0 +1,349 @@
+<tool id="argo_getdata" name="Argo data access" version="@VERSION@" profile="22.05">
+    <description>for global ocean in situ observing system</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="argo_requirements">
+    </expand>
+    <required_files>
+        <include type="literal" path="argo_getdata.py"/>
+        <include type="literal" path="macro.xml"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[ 
+        python $__tool_directory__/argo_getdata.py
+            --user=$user
+             #if $selection.data_selection == 'space_time':
+              --cardinal_1=$selection.cardinal_1
+              --cardinal_2=$selection.cardinal_2
+              --cardinal_3=$selection.cardinal_3
+              --cardinal_4=$selection.cardinal_4
+              --pressure_1=$selection.pressure_1
+              --pressure_2=$selection.pressure_2
+              --date_1=$selection.date_1
+              --date_2=$selection.date_2
+            #else if $selection.data_selection == 'floats':
+              --wmo=$selection.wmo
+            #else:
+              --wmo=$selection.wmo
+              --profile=$selection.profile
+            #end if
+            #if $dataset.data_dataset == 'bgc' :
+              --params=$dataset.params
+              --measured=$dataset.measured
+            #end if 
+            --output_argo=$output_argo
+        ]]>
+    </command>
+    <inputs>
+        <param name="user" type="select" label="Which kind of user are you ?">
```

Comment:
For the label ... do you really want to ask for "What user are you?" 

The option indicate more a question `We have preconfigured some mode of operations for you. What mode do you want to use?`